src/trunk.c

```diff
@@ -8025,14 +8145,21 @@ trunk_print_subtree(trunk_handle        *spl,
    if (hdr->height != 0) {
       for (uint32 i = 0; i < trunk_num_children(spl, node); i++) {
          trunk_pivot_data *data = trunk_get_pivot_data(spl, node, i);
-         trunk_print_subtree(spl, data->addr, log_handle);
+         trunk_print_subtree(log_handle, spl, data->addr);
       }
    }
    cache_unget(spl->cc, node);
 }
 
+/*
+ * trunk_print_memtable() --
+ *
+ * Print the currently active Memtable, and the other Memtables being processed.
+ * Memtable printing will drill-down to BTree printing which will keep
+ * recursing. So ... you'll end up with TONS of output; all very useful.
```

Comment:
Kill this last sentence.

I agree this isn't a useful function in general and we should consider just removing it.

Reply:
Let's retain the function, I like to see all the internals printed, at least as an educational tool.  I think it's good to have such tools that dump everything out.

I deleted the cautionary comment.